.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules
 .idea/
+package-lock.json
+npm-debug.log
+src/ol-layerswitcher_ORIGINAL.js
```

Comment:
Can you please remove `src/ol-layerswitcher_ORIGINAL.js` from `.gitignore` and make sure the file doesn't get committed.

Reply:
Of course! I think I just messed around a bit with some files. Will clear that!